types/lodash/index.d.ts

```diff
@@ -10,7 +10,7 @@
 //                 Georgii Dolzhykov <https://github.com/thorn0>,
 //                 Jack Moore <https://github.com/jtmthf>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.2
+// TypeScript Version: 2.6
```

Comment:
Why does this require typescript 2.6? We still support typescript 2.2, don't we?

Reply:
Oh I see, because `interface WeakSet<T extends object> { }` is not compatible with old versions of typescript. It seems a shame to lose TS <= 2.5 compatibility just because of this change, but I can't think of a great alternative.

We could do this:

```
// @ts-ignore
interface WeakSet<T> { }
```

Then we maintain compatibility with TS <= 2.5, but then we won't use quite the right interface for people who are targeting ES5.

Reply:
Thanks!